src/utils.ts

```diff
@@ -225,3 +225,12 @@ export function copyWithBigint<T>(object: unknown): T {
         )
     );
 }
+
+export function getImageTagByArch(tag: string): string {
+    // Check CPU architecture
+    const arch = process.arch;
+    if (arch === "arm64") {
+        tag = tag.endsWith("-arm") ? tag : `${tag}-arm`;
```

Comment:
Instead of using the ternary operator, it could be checked if arch is arm64 and if tag doesn't end with -arm in the same if-clause.

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/123/commits/be0ea87ac6db94e860ea92f56e799b16af6562dc